Lib/traceback.py

```diff
@@ -1106,6 +1106,10 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
             suggestion = _compute_suggestion_error(exc_value, exc_traceback, wrong_name)
             if suggestion:
                 self._str += f". Did you mean: '{suggestion}'?"
+        elif exc_type and issubclass(exc_type, ModuleNotFoundError) and \
+                sys.flags.no_site:
+            self._str += ". Site initialization is disabled, did you forget to add the \
+                site-package directory to sys.path?"
```

Comment:
Traceback message has too many spaces in the middle:

```
ModuleNotFoundError: No module named 'foo'. Site initialization is disabled, did you forget to add the                 site-package directory to sys.path?
```

Reply:
It should be fixed now, please test it again